lib/modules/dosomething/dosomething_rogue/dosomething_rogue.cron.inc

```diff
@@ -56,9 +56,9 @@ function dosomething_rogue_retry_failed_reportbacks() {
       } else {
         $data = [
           [
-            'rogue_reportback_item_id' => $task->rogue_item_id,
+            'rogue_event_id' => $task->rogue_event_id,
             'status' => $task->status,
-            'reviewer' => $task->reviewer,
+            'event_type' => 'post_photo',
```

Comment:
wait sorry I spoke too soon! This actually has to be `post_photo`  because that is a validation rule on Posts in Rogue. To fix this though, I just [hardcoded this](https://github.com/DoSomething/rogue/pull/112/commits/27cdefcfa9f6288215e0989f4c23f0b004ed009b) in on the Rogue side to save this as a `review` `event_type`.